### ISSUE TITLE
NMM keep asking for "new version setup"

### ISSUE THREAD

**Issue by DTLiem** [Nexus-Mods/Nexus-Mod-Manager#1252]
![bf3e0dff1e8d277d8c0257db4ad9094b](https://user-images.githubusercontent.com/70763235/204090735-7e961902-70e2-4da0-8429-37f4ecd4411b.png)

My NMM version is the latest one, somehow this notification suddenly popping up yesterday, so I cant use the mod manager because if I press cancel it close the program too, eventhough the day before it was normal, I tried to downgrade to 0.84.0 and it still asking the same, do I really need to press Yes? I dont want to reinstall my heavy mods since years ago :(

**Comment by DuskDweller** [comment]
Uninstall NMM removing config files when asked. Then when you reinstall it make TRIPLE sure you're selecting the correct (the ones you were using before) Install Info, Mods, VirtualInstall folders.

**Comment by DTLiem** [comment]
Hello
Already did everything you asked, but unfortunately the notification still persist.

**Comment by DuskDweller** [comment]
Did you change the drive where your game is installed?

**Comment by DTLiem** [comment]
No, the NMM always installed on C while the game mods, install info, and virtualinstall folder are on D, been like that for years

**Comment by DuskDweller** [comment]
NMM prompts you to migrate your stuff when it can't find info about your virtual install files.

Make sure there's a VirtualModConfig.xml file in your VirtualInstall folder, and make sure NMM is set to work with that folder. Sometimes you're 100% it's correct, and then you notice it is pointing to another drive but same path.

**Comment by DTLiem** [comment]
VirtualModConfig.xml file is there inside VirtualInstall folder (60mb size), and NMM is already set on that folder, and in the end still asking the same prompts

**Comment by susistuart** [comment]
I have the same problem after the most recent update to 0.84.3.0. I have not changed anything in years about where I have stored my game, mods, or NMM files; however, NMM switched the paths for Skyrim SE after this update. I corrected the paths in the user.config file in appdata, which did not resolve the issue. 

In the end, I clicked "no", stopped the process immediately, reinstalled the one mod it uninstalled, and am living my life with a bunch of red boxes--they'll all get updated/reinstalled someday, anyway. 

It ONLY affected Skyrim SE. Every other game seems to be fine.

**Comment by Minolta50** [comment]
same issue. reinstalling program did not help
I don't want to delete all my mods!
any solution?
![2024-01-04_211557](https://github.com/Nexus-Mods/Nexus-Mod-Manager/assets/132227788/0b14ef04-4dd2-494e-bb42-706e0d4f6eb1)

**Comment by Minolta50** [comment]
> та же проблема. переустановка программы не помогла, не хочу удалять все свои моды! любое решение? ![2024-01-04_211557](https://private-user-images.githubusercontent.com/132227788/294236612-0b14ef04-4dd2-494e-bb42-706e0d4f6eb1.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDQzODUzNDUsIm5iZiI6MTcwNDM4NTA0NSwicGF0aCI6Ii8xMzIyMjc3ODgvMjk0MjM2NjEyLTBiMTRlZjA0LTRkZDItNDk0ZS1iYjQyLTcwNmUwZDRmNmViMS5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMTA0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDEwNFQxNjE3MjVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04ZmI2MWY2NWM3YjhmZDg4OGYwMmJkOTVkMTkwMDAwMjFiYWFlYzQ4ODExMzg3ODRlYzJjZGQ2ZDkyNjQ5ZmI5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.3jwJAj98oxgZhlQmNqHpqHEmyU3Pme8W_1x4zShjt0Y)

I found a solution. Perhaps it will help someone.
In my case, for some reason the uninstaller did not delete user config. Accordingly, NMM worked with out of date paths for the VirtualInstall directory.
1. Uninstall NMM and delete file "user.config" in path ...AppData\Local\Black_Tree_Gaming\NexusClient.exe_Url_dbufbdytrwwvowz4esvbbxuf50zn22jd\etc
2. Install NMM and choose actual paths for all directories.

**Comment by jaudi5k** [comment]
I was able to get around this by
1. Pulling up Ctrl+alt+del 
2. Press "No" for the forced update
3. Press Cancel while it tries to start uninstalling your mods
4. Use the Ctrl+alt+del screen to get out of nexus mod manager
5. Relaunch Nexus Mod Manager
